keanu-python/tests/test_const.py

```diff
@@ -0,0 +1,153 @@
+import keanu as kn
+import numpy as np
+import pytest
+
+def test_const_takes_int_numpy_tensor():
```

Comment:
Optional: you could use parametrized fixtures to combine 3 tests into 1.

https://docs.pytest.org/en/latest/parametrize.html

Reply:
This is cool!